### PR TITLE
Initial layout redesign

### PR DESCRIPTION
**Pull request by peteromano**

### CODE REVIEW

**Comment by peteromano** [comment]
screen shot 2016-10-09 at 3 04 17 pm
screen shot 2016-10-09 at 3 04 29 pm

**Comment by peteromano** [comment]
screen shot 2016-10-09 at 3 54 40 pm
screen shot 2016-10-09 at 3 54 47 pm
screen shot 2016-10-09 at 3 54 54 pm
screen shot 2016-10-09 at 3 54 57 pm

**Comment by peteromano** [comment]
screen shot 2016-10-09 at 4 00 09 pm
screen shot 2016-10-09 at 4 00 17 pm
screen shot 2016-10-09 at 4 00 22 pm
screen shot 2016-10-09 at 4 00 26 pm

File: src/components/InfoBar/InfoBar.js [hunk 1]
```diff
@@ -16,14 +16,12 @@ export default class InfoBar extends Component {
     const {info, load} = this.props; // eslint-disable-line no-shadow
     const styles = require('./InfoBar.scss');
     return (
-      <div className={styles.infoBar + ' well'}>
-        <div className="container">
-          This is an info bar
-          {' '}
```
Comment (jshor) [inline src/components/InfoBar/InfoBar.js]: is this to force a space between this and the `<strong>`?

File: src/containers/Home/Home.scss [hunk 2]
```diff
@@ -7,21 +7,37 @@
...
   }
 }
 
-.masthead {
+.home .masthead {
   background: #292929;
   color: white;
   text-align: center;
-  padding: 2em 0;
+  display: flex;
```
Comment (jshor) [inline src/containers/Home/Home.scss]: 💯

File: src/components/InfoBar/InfoBar.js (hunk 1 above)
Comment (peteromano) [inline src/components/InfoBar/InfoBar.js]: No idea. Good question.. must be some caveat here. This InfoBar stuff is part of the example boilerplate.

File: src/containers/Home/Home.scss (hunk 2 above)
Comment (peteromano) [inline src/containers/Home/Home.scss]: Yea, the masthead container stretches to the viewport height using `height: 94vh`, but need flex box to actually vertically center the content.

Never heard of vertical rhythm. Nice share; I'll check it out!

**Pull request merged**